rlang/src/main/java/org/apache/zeppelin/r/IRInterpreter.java

```diff
@@ -152,6 +153,17 @@ protected void initIRKernel() throws IOException, InterpreterException {
     }
   }
 
+  @Override
+  protected Map<String, String> setupKernelEnv() throws IOException {
+    Map<String, String> envs = super.setupKernelEnv();
+    String pathEnv = envs.getOrDefault("PATH", "");
+    if (condaEnv != null) {
+      pathEnv = new File(".").getAbsolutePath() + "/" + condaEnv + "/bin:" + pathEnv;
```

Comment:
Please use `File.separator` and `File.pathSeparator` to build the `PATH` environment variable.

Reply:
Fixed

Reply:
Looks weirder than I thought, but I think that should be correct.